src/runtime_src/tools/xclbinutil/Section.cxx

```diff
@@ -88,29 +87,29 @@ Section::registerSectionCtor(enum axlf_section_kind _eKind,
                              Section_factory _Section_factory) {
   // Some error checking
   if (_sKindStr.empty()) {
-    std::string errMsg = XUtil::format("ERROR: Kind (%d) pretty print name is missing.", _eKind);
-    throw std::runtime_error(errMsg);
+    auto errMsg = boost::format("ERROR: Kind (%d) pretty print name is missing.") % (unsigned int) _eKind;
+    throw std::runtime_error(errMsg.str());
   }
 
   if (m_mapIdToName.find(_eKind) != m_mapIdToName.end()) {
-    std::string errMsg = XUtil::format("ERROR: Attempting to register (%d : %s). Constructor enum of kind (%d) already registered.",
-                                       (unsigned int)_eKind, _sKindStr.c_str(), (unsigned int)_eKind);
-    throw std::runtime_error(errMsg);
+    auto errMsg = boost::format("ERROR: Attempting to register (%d : %s). Constructor enum of kind (%d) already registered.")
+                                %(unsigned int)_eKind % _sKindStr % (unsigned int)_eKind;
```

Comment:
F.Y.I. - Future PR will be beautified against the google coding style (which is what XRT has adopted).   Didn't do so here simply because it would overwhelm the code reviewer with all of the changes.